app/questionnaire/schema_utils.py

```diff
@@ -18,9 +18,7 @@ def find_pointers_containing(
         if search_key in input_data:
             yield pointer or ""
         for k, v in input_data.items():
-            if (isinstance(v, dict)) and search_key in v:
-                yield pointer + "/" + k if pointer else "/" + k
-            elif isinstance(v, (list, tuple, dict)):
+            if isinstance(v, (list, tuple, dict)):
```

Comment:
I've done this because in the instance you have something like
`{"question": {"id": "...", "answers": [{"id": }]}}`
it'll see that the search key is in question, and yield the pointer, and not then go through the answers
now will do both

Reply:
Good spot. Found out the hard way? 💥

Reply:
just a bit 😄